scripts/deploy

```diff
@@ -17,13 +17,15 @@ function update_coredns_configmap() {
         sed 's/dnsip/'${CLUSTER_IP}'/g' scripts/resources/coredns-cm.yaml > /tmp/coredns-cm-${CLUSTER_IP}.yaml
         kubectl -n kube-system replace -f /tmp/coredns-cm-${CLUSTER_IP}.yaml
         kubectl -n kube-system describe cm coredns
-    fi
+    else 
+        run_subm_clusters install_service_import
```

Comment:
Why is `install_service_import` now conditional on the coredns CM?

Reply:
The changes to install CRD is done via the following PRs 

Operator
https://github.com/submariner-io/submariner-operator/pull/550
Helm
https://github.com/submariner-io/submariner-charts/pull/51

While shipyard get  the latest from helm the operator changes are picked up only when we update subctl version in [1] is updated.
https://github.com/submariner-io/shipyard/blob/master/package/Dockerfile.shipyard-dapper-base#L47 

The "if" above identifies a helm based installation where core-dns map needs to be updated explicitly. So I added an else for doing things specific to operator installation. This shall be removed when the latest subctl is available.